ArjunaCore/arjuna/classes/com/arjuna/ats/arjuna/logging/arjunaI18NLogger.java

```diff
@@ -1629,6 +1629,15 @@ public void warn_objectstore_JDBCImple_over_max_image_size(int imageSize,
 			format = MESSAGE_FORMAT)
 	@LogMessage(level = INFO)
 	public void info_maxIO();
+
+	@Message(id = 12406, value = "Invalid class type in system property ObjStoreBrowserHandlers: ''{0}''", format = MESSAGE_FORMAT)
+	@LogMessage(level = WARN)
+	void warn_invalidObjStoreBrowser_type(String type, @Cause Exception e);
```

Comment:
@mmusgrov I don't think this is used in the code? But I will approve it as it keeps the numbering consistent. Was that why it was added? If you would like to change something as a result of my comment and need my review again please can you re-request a review?

Reply:
This is a cherry pick - I validated the results by running the testsuite, ie I did not analyse/eyeball the logic.

Reply:
Plus make too many changes by eyeballing the code is going to cause problems with future cherry-picks.

Reply:
It's a bit strange because that message was added on main (and 6.0) by https://github.com/jbosstm/narayana/commit/97211e69d2acd14b3e0df64ce24d9c7401dc4b75 - maybe it's just how cherry-pick works then.

Anyway, it is fine with me that we include it because otherwise the numbering could be a bit harder to understand